src/qt/forms/debugwindow.ui

```diff
@@ -652,12 +652,12 @@
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
-           <property name="placeholderText">
```

Comment:
I guess the `property` order change has been done through QTCreator? I would have probably unstaged those.